tox.ini

```diff
@@ -58,8 +58,7 @@ deps =
     psutil==5.7.0
     setuptools==59.6.0
     protobuf>=3.20,<=3.20.1
-    apduboy>=0.5.0
-    
+    git+https://github.com/LedgerHQ/apduboy.git@d167f620430afeed313edc9fc1315d6d9c50234e
```

Comment:
@solarw 
Did the solana dependency get into this tox as well? If so, we was to ensure that is properly contained in the plugin folder